app/node_modules/petition/components/state-info.js

```diff
@@ -35,7 +35,9 @@ export default React.createClass({
   },
 
   componentWillReceiveProps(nextProps) {
-    this.fetchTransitions(nextProps.petition.state);
+    if (nextProps.state !== this.props.state) {
```

Comment:
remove if statement, do not delete error in fetchTransitions (https://github.com/iris-dni/iris-admin/blob/edd005e953d8624260dd42726306b45db29aec12/app/node_modules/petition/components/state-info.js#L45) instead